_posixsubprocess_helpers.c

```diff
@@ -2,6 +2,12 @@
    This file is #included by _posixsubprocess.c and the functions
    are declared static to avoid exposing them outside this module. */
 
+/* _posixsubprocess_config.h was already included by _posixsubprocess.c
+ * which is #include'ing us despite the .c name.  HAVE_SIGNAL_H comes
+ * from there.  Yes, confusing! */
```

Comment:
Yes this is confusing. If this module is not compiled separately but included by _posixsubprocess.c, why not name this _posixsubprocess_helpers.c? I guess we keep it as is because this is the way it was in upstream?

Reply:
Upstream CPython doesn't have this file, the helpers file contains things that are built in to the Python 3 interpreter.